File: problem0001/java/Problem0001.java
/**
 * Problem: http://projecteuler.net/index.php?section=problems&id=1
 * Author: Elie De Brauwer <elie @ de-brauwer.be>
 * License: Simplified BSD
 * Date: December 11, 2010
 */

class Problem0001
{
    public static void main(String [] args)
    {
	int total = 0;
	for (int i = 0; i < 1000; i++)
	{
	    if ((i%3 == 0) || (i%5 == 0))
	    {
		total += i;
	    }
	}
	System.out.println("Total: " + total);
    }
}

File: problem0001/haskell/problem0001.hs
{-
Problem: http://projecteuler.net/index.php?section=problems&id=1
Author: Elie De Brauwer <elie @ de-brauwer.be>
License: Simplified BSD
Date: December 11, 2010
-}

problem0001 :: Integer
problem0001 = sum [ x | x <- [0..999], x `mod` 5 == 0 ||   x `mod` 3 == 0  ]

main = print problem0001